presentation/src/main/java/com/anytypeio/anytype/presentation/spaces/SpaceSettingsViewModel.kt

```diff
@@ -325,6 +330,41 @@ class SpaceSettingsViewModel(
         }
     }
 
+    fun onSpaceImagePicked(path: String) {
+        viewModelScope.launch {
+            uploadFile.async(
+                params = UploadFile.Params(
+                    path = path,
+                    space = params.space,
+                    type = Block.Content.File.Type.IMAGE
+                )
+            ).fold(
+                onSuccess = { file ->
+                    setSpaceDetails.async(
```

Comment:
better to move setSpaceDetails to an another function

Reply:
done